libcontainer/rootfs_linux.go

```diff
@@ -202,10 +202,19 @@ func prepareRootfs(pipe *syncSocket, iConfig *initConfig) (err error) {
 		return err
 	}
 
-	if config.NoPivotRoot {
-		err = msMoveRoot(config.Rootfs)
-	} else if config.Namespaces.Contains(configs.NEWNS) {
+	if config.Namespaces.Contains(configs.NEWNS) {
 		err = pivotRoot(config.Rootfs)
+		if config.NoPivotRoot {
+			logrus.Warnf("--no-pivot is deprecated and may be removed or silently ignored in a future version of runc -- see <https://github.com/opencontainers/runc/issues/4435> for more details")
```

Comment:
Because this cmd flag is deprecated, so need to update this flag’s help text for ‘runc create/ run’ as well?

https://github.com/opencontainers/runc/blob/3563055cc765c78b333a41e2acdcda095d04a2a6/create.go#L46-L48